index.html

```diff
@@ -36,237 +36,257 @@ <h4>My Logo</h4>
-            </a>
-          </li>
-        </ul>
-      </div>
-      <div id="iteme-7"><button class="buttonresume">Get my Resume</button></div>
-      <div class="my-accordion">
-        <ul>
-          <li class='menu'>Languages
+    <div class="social-icons">
+      <ul class="social-icons">
+        <li class="social-icon">
+          <a href="https://twitter.com/ifnotlily"> <img src="images/twitter.svg" alt="Lilian's Twitter" /> </a>
+        </li>
+        <li class="social-icon">
+          <a href="https://www.linkedin.com/in/lilian-francis/">
+            <img src="images/Linkedin.svg" alt="Lilian's LinkedIn" />
+          </a>
+        </li>
+        <li class="social-icon">
+          <a href="https://medium.com/@chatwitlily/">
+            <img src="images/medium.svg" alt="Lilian's Medium" /> </a>
+        </li>
+        <li class="social-icon">
+          <a href="https://angel.co/u/lilian-francis"> <img src="images/angellist.svg" alt="Lilian's AngelList" />
+          </a>
+        </li>
+        <li class="social-icon">
+          <a href="https://github.com/ifnotlily/">
+            <img src="images/github.svg" alt="Lilian's Github" />
+          </a>
+        </li>
+      </ul>
+    </div>
+    <div id="iteme-7"><button class="buttonresume">Get my Resume</button></div>
+    <div class="my-accordion">
+      <ul>
+        <li class='menu'>Languages
 
 
-          <li class='submenu'><a href=""><img src="/images/Ellipse1.png" alt="javascript">JavaSript</a></li>
-          <li class='submenu'><a href=""><img src="/images/Ellipse2.png" alt="html">HTML</a></li>
-          <li class='submenu'><a href=""><img src="/images/Ellipse3.png" alt="css">CSS</a>
-          </li>
+        <li class='submenu'><a href=""><img src="/images/Ellipse1.png" alt="javascript">JavaSript</a></li>
+        <li class='submenu'><a href=""><img src="/images/Ellipse2.png" alt="html">HTML</a></li>
+        <li class='submenu'><a href=""><img src="/images/Ellipse3.png" alt="css">CSS</a>
+        </li>
 
 
-          <li class='menu'>Frameworks<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
+        <li class='menu'>Frameworks<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
 
 
-          <li class='menu'>Skills<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
-        </ul>
+        <li class='menu'>Skills<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
+      </ul>
 
+    </div>
 
-      </div>
+  </section>
+  <section class="contactform">
+    <div class="container"></div>
+    <div class="content-form">
+      <h3>Contact me</h3>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it </p>
+      <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/fs-frm"
```

Comment:
- [ ] It looks like your link for form spree is broken. To use this service you need to subscribe on the page, then create a form inside it and add the link they give to you. Test the form before asking for another code review (the test should work, you should receive an email where the form info appears). 🌵 
![image](https://user-images.githubusercontent.com/20563252/126365642-f30221d1-2828-4266-aa06-2261b1c4937a.png)